apps/strapi-cms/src/components/sections/cards.json

```diff
@@ -0,0 +1,36 @@
+{
+  "collectionName": "components_sections_cards",
+  "info": {
+    "displayName": "Cards",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "theme": {
+      "type": "enumeration",
+      "enum": [
+        "light",
+        "dark"
+      ],
+      "default": "light",
+      "required": true
+    },
+    "title": {
+      "type": "string",
+      "required": true
+    },
+    "subtitle": {
+      "type": "string"
+    },
+    "body": {
+      "type": "richtext"
```

Comment:
after [what discovered in another PR](https://github.com/pagopa/b2b-portals/pull/130#discussion_r1466175920), should we change this to `text`?

Reply:
I don't think we should for now.

That issue has only presented itself for the BannerLink component (Editorial does the same and is fine for instance, Footer as well) so there's a good chance it's an isolated issue.